cmd/helm/release_testing.go

```diff
@@ -18,28 +18,39 @@ package main
 
 import (
 	"io"
+	"time"
 
 	"github.com/spf13/cobra"
 
+	"helm.sh/helm/cmd/helm/require"
 	"helm.sh/helm/pkg/action"
 )
 
 const releaseTestHelp = `
-The test command consists of multiple subcommands around running tests on a release.
-
-Example usage:
-    $ helm test run [RELEASE]
```

Comment:
Currently `helm test run <TAB>` will provide dynamic completion by listing available releases.

If you remove the `run` subcommand, could you update the dynamic completion code by changing the string `helm_test_run` to `helm_test` in `cmd/helm/root.go`?

Thank you.

Reply:
Nice catch @marckhouzam, updated.